plaaces-to-go/src/styles/Menu.module.css

```diff
@@ -0,0 +1,101 @@
+.navbar{
+    display: flex;
+    align-items: center;
+    background: #f7e9da;
+}
+
+.navLogo{
+    display: flex;
+    flex: 1;
+}
+
+.txtItem{
+    color: #3f6b7c;
+    font-size: 20px;
+    padding: 20px;
+    font-family: 'Lato', 'Arial Narrow', Arial, sans-serif;
```

Comment:
You want this font for the whole project? Try moving this style to the globals.css instead

Reply:
I put this font in the global file :D Thanks

Reply:
Then you can delete this line